components/brave_wallet_ui/common/slices/endpoints/wallet.endpoints.ts

```
@@ -131,6 +131,11 @@ export const walletEndpoints = ({
           const { keyringService } = api
 
           const result = await keyringService.createWallet(arg.password)
+          if (!result.mnemonic) {
+            return {
+              error: 'Unable to create wallet'
+            }
```

let's throw an error and let the catch block return the error message